packages/@vuepress/theme-default/src/client/styles/vars.scss

```diff
@@ -65,6 +65,18 @@
   --sidebar-width-mobile: calc(var(--sidebar-width) * 0.82);
   --content-width: 740px;
   --homepage-width: 960px;
+
+  // transition vars
+  --t-duration: 0.3s;
+  --t-duration-slide: 0.3s;
+  --t-all: all var(--t-duration);
+  --t-opacity: opacity var(--t-duration);
+  --t-color: color var(--t-duration);
+  --t-border-color: border-color var(--t-duration);
+  --t-background-color: background-color var(--t-duration);
+  --t-color-all: var(--t-color), var(--t-border-color),
+    var(--t-background-color);
+  --t-transform: transform var(--t-duration);
```

Comment:
Seems that transition-timing-function is not supported?

Reply:
https://github.com/vuepress/vuepress-next/pull/325/commits/74ff30470f4631ccc0f199f4ad688c62a653cb2b